WeatherStormPower/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using WeatherStormPower.Models.WeatherBots;
+using WeatherStormPower.Models.DataParsing;
+using WeatherStormPower.Models;
+
+namespace WeatherStormPower;
+
+public class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Enter the Json File Path for the Bot Configuration: ");
+        var FilePath = Console.ReadLine();
+        var configurations = BotConfigurationHandler.LoadConfiguration(FilePath);
+        
+        Console.WriteLine();
+
+        Console.WriteLine("Enter weather data (in JSON format or XML format.");
+        string input = Console.ReadLine()+ Console.ReadLine()+Console.ReadLine()+ Console.ReadLine() + Console.ReadLine();
+
+        Console.WriteLine() ;
+
+        IWeatherDataParser parser;
+        if (input.TrimStart().StartsWith("{"))
+            parser = new JsonWeatherDataParsing();
+        else
+            parser = new XmlWeatherDataParsing();
+
+        var weatherMonitor = new WeatherMonitor();
+        if (configurations["RainBot"].Enabled)
```

Comment:
same here, think about a design pattern.

Reply:
and this whole logic here in program is business logic that can be honored in a different place, take that into account when thinking about the design patterns.